lighthouse-core/gather/drivers/driver.js

```diff
@@ -473,9 +473,22 @@ class Driver {
     const origin = `${parsedURL.protocol}//${parsedURL.hostname}` +
       (parsedURL.port ? `:${parsedURL.port}` : '');
 
+    // Clear all types of storage except cookies, so the user isn't logged out.
+    const storageTypes = [
+      'appcache',
+      // 'cookies',
+      'file_systems',
+      'indexeddb',
+      'local_storage',
+      'shader_cache',
+      'websql',
+      'service_workers',
+      'cache_storage'
+    ].join(',');
+
     return this.sendCommand('Storage.clearDataForOrigin', {
       origin,
-      storageTypes: 'all',
+      storageTypes,
```

Comment:
since this param name (and `origin`) is API and unchangeable, let's not use the shorthand.

```js
origin: origin,
storageTypes: typesToClear
```